src/app/containers/App/index.jsx

```diff
@@ -9,7 +9,7 @@ import App from './App';
 
 export const ClientApp = props => (
   <BrowserRouter {...props}>
-    <App initialData={props.data} routes={props.routes} />
+    <App initialData={props.data} routes={props.routes} headers={props.data.headers} />
```

Comment:
I see that we're passing `props.data` as prop already, can't we get `headers` from `prop.initialData.headers` from the component ?

Reply:
Agree with Rowland here, is there any need for the extra `headers` prop?

Reply:
Fixed!